go/client/identify_ui.go

```diff
@@ -23,30 +23,6 @@ func NewIdentifyTrackUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
 	return keybase1.IdentifyUiProtocol(&IdentifyUIServer{ui})
 }
 
-func cliIsLoggedIn(g *libkb.GlobalContext) bool {
-	configCli, err := GetConfigClient(g)
-	if err != nil {
-		return false
-	}
-
-	currentStatus, err := configCli.GetCurrentStatus(context.TODO(), 0)
-	if err != nil {
-		return false
-	}
-
-	return currentStatus.LoggedIn
-}
-
-// NewIdentifyMaybeTrackUIProtocol returns either IdentifyUiProtocol with
-// IdentifyTrackUI or IdentifyUI depending if user is logged in or not.
-func NewIdentifyMaybeTrackUIProtocol(g *libkb.GlobalContext) rpc.Protocol {
```

Comment:
My memory of this isn't crystal clear, but I believe this was in place to allow `keybase pgp encrypt` to work logged out.  Does it still work logged out?

    keybase pgp encrypt user -m "hi" --no-self

works for me logged out in production currently.